From 81fbc54d293ba044515a39b4a6ecb69bf565b694 Mon Sep 17 00:00:00 2001
From: Wenyou Yang <wenyou.yang@atmel.com>
Date: Tue, 27 Jan 2015 17:03:32 +0800
Subject: [PATCH] 1-Wire: add the information for the SAMA5D3X_CMP board

Add information for the SAMA5D3X_CMP board, its 1-Wire board name is "SAMA5D36-CP".

Signed-off-by: Wenyou Yang <wenyou.yang@atmel.com>
---
 driver/board_hw_info.c  | 1 +
 include/board_hw_info.h | 1 +
 2 files changed, 2 insertions(+)

diff --git a/driver/board_hw_info.c b/driver/board_hw_info.c
index 5c413e4..6562eeb 100644
--- a/driver/board_hw_info.c
+++ b/driver/board_hw_info.c
@@ -98,6 +98,7 @@ static struct {
 	{"SAMA5D34-CM",	BOARD_TYPE_CPU,	BOARD_ID_SAMA5D34_CM},
 	{"SAMA5D35-CM",	BOARD_TYPE_CPU,	BOARD_ID_SAMA5D35_CM},
 	{"SAMA5D36-CM",	BOARD_TYPE_CPU,	BOARD_ID_SAMA5D36_CM},
+	{"SAMA5D36-CP",	BOARD_TYPE_CPU,	BOARD_ID_SAMA5D36_CP},
 	{"SAMA5D44-MB",	BOARD_TYPE_EK,	BOARD_ID_SAMA5D4_MB},
 	{"SAMA5D4-XULT",BOARD_TYPE_EK,	BOARD_ID_SAMA5D4_MB},
 	{0,		0,		0},
diff --git a/include/board_hw_info.h b/include/board_hw_info.h
index 2aba80f..f8cd025 100644
--- a/include/board_hw_info.h
+++ b/include/board_hw_info.h
@@ -47,6 +47,7 @@
 #define	BOARD_ID_PDA7_DM	15
 #define	BOARD_ID_SAMA5D4_MB	16
 #define	BOARD_ID_SAMA5D4_XULT	17
+#define	BOARD_ID_SAMA5D36_CP	18
 
 /* Vendor id definition */
 #define VENDOR_EMBEST		1
-- 
1.9.1

